pulsar-client-admin/src/main/resources/mltool/ManagedLedgerAdminTool.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
+
+import argparse
+import traceback
+
+from kazoo.client import KazooClient
+
+from mltool import MLDataFormats_pb2
+
+'''
+This util provides api to access managed-ledger data if it's ONLY stored in binary format in to zookeeper.
+It also provides command line tool-access to execute these commands.
+
+Run: run from parent directory
+python -m mltool.ManagedLedgerAdminTool
+'''
+managedLedgerPath = "/managed-ledgers/"
+printMlCommand = "print-ml"
+deleteMlLedgerIds = "delete-ml-ledgers"
+printCursorsCommands = "print-cursor"
+updateMakDeleteCursor = "update-mark-delete-cursor"
+
+'''
+Returns managed-ledger info for given managed-leger path 
+Parameters
+----------
+zk : KazooClient
+   Zookeeper-client instance to query zk-client.
+mlPath : str
+    managed-ledger path
+''' 
+def getManagedLedgerInfo(zk, mlPath):
+    try:
+        # get managed-ledger info
+        mlData = zk.get(mlPath)[0]
+        mlInfo = MLDataFormats_pb2.ManagedLedgerInfo()
+        mlInfo.ParseFromString(mlData)
```

Comment:
yes, earlier I thought we will use this tool after merging PR which forces to write in Binary format. However, I have added Text support also.